SPECS/libtar/libtar.spec

```diff
@@ -1,16 +1,24 @@
 Summary:        C library for manipulating tar files
 Name:           libtar
 Version:        1.2.20
-Release:        8%{?dist}
-URL:            https://github.com/tklauser/libtar/
+Release:        9%{?dist}
 License:        BSD
-Group:          System Environment/Libraries
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          System Environment/Libraries
+URL:            https://github.com/tklauser/libtar/
 #Source0:       https://github.com/tklauser/%{name}/archive/v%{version}.tar.gz
 Source0:        %{name}-%{version}.tar.gz
+# This patch appears to replicate Fedora's ' libtar-1.2.11-bz729009.patch'
 Patch0:         libtar-gen-debuginfo.patch
-patch1:         libtar-CVE-2013-4420.patch
+# This patch appears to fix the same issue as Fedora's 'libtar-1.2.20-no-static-buffer.patch'
+Patch1:         libtar-CVE-2013-4420.patch
+# CVE patches + other fixes from Redhat
+Patch2:         libtar-1.2.11-mem-deref.patch
+Patch3:         libtar-1.2.20-CVE-2021-33643-CVE-2021-33644.patch
+Patch4:         libtar-1.2.20-CVE-2021-33645-CVE-2021-33646.patch
+Patch5:         libtar-1.2.20-fix-resource-leaks.patch
+Patch6:         libtar-1.2.20-static-analysis.patch
 Provides:       libtar.so.0()(64bit)
```

Comment:
I believe RPM should autogenerate this provides- if you have the time, would you mind removing this line and testing that hypothesis? Manual provides like this are 🚩for me.

Reply:
I'll do a follow up PR for that. This is a critical CVE and I'd rather get it rolling out the door asap